Assets/Scripts/EntityManager.cs

```diff
@@ -5,28 +5,31 @@
 public class EntityManager : MonoBehaviour
 {
 	[Header("Properties")]
-	public int maxEnemiesPerRoom = 6;
-	public int minEnemiesPerRoom = 4;
-	public float enemySpawnSeconds = 0.5f;
+	public int minEnemiesPerSpawn = 1;
+	public int maxEnemiesPerSpawn = 3;
+	public float enemySpawnDelay = 5.0f; // The number of seconds between spawn delay.
+	public float enemySpawnIncrease = 0.01f; // The decrease in seconds of the spawn delay every second.
+	public float spawnMargin = 1f;
+	public float spawnSpacing = 1.0f;
+	public float approxEnemyRadius = 1.5f;
+	public float spawnDistance = 5.0f; // The distance over which enemies spawn, none spawn after this distance.
+
 	public GameObject[] enemyPrefabs;
 
 	[Header("References")]
+	public Camera _camera;
```

Comment:
Should this start with _ if it's public?

Reply:
Yes, but then it shadows the built in "camera" variable